dotnet/src/Svg.cs

```diff
@@ -0,0 +1,162 @@
+﻿using Microsoft.AspNetCore.Html;
+using System;
+using System.Collections.Concurrent;
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
+
+namespace StackExchange.StacksIcons;
+
+public static partial class Svg
+{
+    private const int MaxReasonableSize = 4500;
+    public static readonly SvgImage Empty = new(string.Empty);
+
+    public static partial class Spot
+    {
+        /// <inheritdoc cref="Svg.GetImage(string?, bool, bool)"/>
+        internal static SvgImage GetImage([CallerMemberName] string? fileName = null) =>
+            // For spots, we intentionally want to bypass the size check on these rarely used items to make maintenance easier.
+            Svg.GetImage(fileName, isSpot: true, bypassSizeCheck: true);
+    }
+
+    /// <summary>
+    /// A hash that changes when any of the icons change or one gets added, removed, etc. Calculated in GetImage() by replacing this
+    /// value with a hash of the old value combined with the file contents. Thus relies on the fact that C# guarantees in-order execution
```

Comment:
Thus is the most important suggestion ever 😝:

```suggestion
    /// value with a hash of the old value combined with the file contents. This relies on the fact that C# guarantees in-order execution
```

Reply:
Too late, I already removed this property and _thus_ (heh), the comment as well. 😛

Reply:
Well played!